README.md

```diff
@@ -156,3 +156,31 @@ title: Don't repeat yourself
 
 # {{ title }}
 ```
+
+### Shortcodes
+
+Shortcodes are reusable code snippets that allow us to sweep away complicated scripts into a nice, easy user interface.
+
+For example this is the script for a YouTube embed.
+
+```
+<iframe
+  width="560"
+  height="315"
+  src="https://www.youtube.com/embed/vQF17UBU4RE"
+  title="Ceph Tech Talk: Karan Singh - Scale Testing Ceph with 10Billion+ Objects 2020-10-01"
+  frameborder="0"
+  allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+  allowfullscreen
+></iframe>
+```
+
+We can capture this script in a shortcode so we don't need to repeat it everytime we want to embed a YouTube video. It accepts two values, the `id` and the `title` of the video and uses the following syntax.
+
+```
+{% YouTube 'vQF17UBU4RE', 'Ceph Tech Talk: Karan Singh - Scale Testing Ceph with 10Billion+ Objects 2020-10-01' %}
+```
+
+The first argument is the name of the shortcode `YouTube` and then we pass it the id of the video `vQF17UBU4RE` followed by the title `Ceph Tech Talk: Karan Singh - Scale Testing Ceph with 10Billion+ Objects 2020-10-01`.
```

Comment:
```suggestion
We call shortcodes by name with the Nunjucks block syntax: `{% YouTube %}`. The first argument we pass is the `id` of the video (e.g. `'vQF17UBU4RE'`). The second (optional) argument will set the `title` attribute of the iframe embed (e.g. `'Ceph Tech Talk: Karan Singh - Scale Testing Ceph with 10Billion+ Objects 2020-10-01'`).

_Note:_ Shortcode arguments are type-/space-sensitive, so should should include surrounding `'` quote marks.
```

Thanks @Pete-Robelou. Other comments are very minor terminology tweaks. 

This one though I thought could use some more explicit instruction. How's this read? Technically `YouTube` is more like the function/shortcode name, while the first argument is `id`, and second `title` (if we're thinking about the function signature).

Reply:
Looks good.